### DISCUSSION TITLE
Background opacity level

### DISCUSSION THREAD

**Discussion by genesistoxical** [BLumia/pineapple-pictures#63] [Ideas]
Hello! I have found this viewer and it is the best I have tried in many years. I wish it had the option of my old favorite image viewers: background opacity level (by background I mean the one that can be toggled to checkerboard). I feel that it is very dark, it hurts my eyes a little, I would like to add more transparency.

Thank you very much _(and sorry for my English)_.

**Comment by BLumia** [comment]
Hi, do you mean you want a brighter checkerboard background? If you do, you can try holding <kbd>Shift</kbd> and then click the checkerboard button and see if that's what you want :)
